.gitignore

```diff
@@ -2,12 +2,13 @@
 # Node
 
 # Logs
-# logs
+logs
 *.log
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+.pnpm-debug.log*
```

Comment:
est-ce worth de rajouter `pnpm` alors qu'on l'utilise pas ?

Reply:
Oh it's just automatically generated via [gitignore.io](https://www.toptal.com/developers/gitignore), it's a general convention and we normally never edit it directly but with following custom rules when needed.

Reply:
ça marche !